content/docs/contributing.md

```diff
@@ -4,15 +4,17 @@ description: "Contributing to Open Source"
 draft: false
 ---
 
-Participation in open source projects is highly encouraged, and the Open Source Program Office wants to make this process as simple as possible. All Saucers are expected to follow common sense regarding the type of information and code is contributed to upstream open source projects. If you are in doubt, please reach out to the OSPO or ask your manager for direction.
+Participation in open source projects is highly encouraged, and the Open Source Program Office wants to make this process as simple as possible. Saucers are responsible to preserve our valuable information and code. OSPO and your manager can help you understand and determine what is open sourceable and what must remain private. In doubt, always ask.
 
 ### Summary
 
-We highly encourage contributions to existing open source projects. There are some requirements before you start patching, and guides to help you figure out the right path:
+We highly encourage contributions to [existing open source projects](https://opensource.saucelabs.com/projects/).
```

Comment:
The original phrase was meant to say that OSPO encourages contributions to any project that is open source, given that the guidelines are respected. 
The new phrase suggests contributing only to the open source projects featured in our OSS website.

Reply:
What do you think about this phrase? The idea behind this change was to easier the path between the wish and a project :)

We highly encourage contributions to any project that is open source, given that the guidelines are respected. Don't know where to start? Here you can find [Sauce Labs open source projects](https://opensource.saucelabs.com/projects/) and here you can find a list of [Open Source projects which support underrepresented groups](https://opensourcediversity.org/#projects) and where you can participate.

Reply:
I'd prefer more something along: "We highly encourage contributions to any open source project that aligns with the guidelines. If you are looking for projects to contribute, here are some suggestions: ...", and then a list of links of projects that we want to suggest.